### ISSUE TITLE
Trouble parsing query string on default route

### ISSUE THREAD

**Issue by lbssousa** [krasimir/navigo#96]
I'm trying to parse a query string passed to my default route using navigo. Example URL:
```
http://myhost/myproject?check=1
```

Example code:
```
var router = new Navigo('http://myhost/myproject', false, '#!');

router.on(function (query) {
    console.log('Query string: ' + query);
}).resolve();
```

If I initialize my router with `useHash = false`, then I'm unable to parse the query `check=1` (the `query` parameter in the handler function above only returns an empty string). However, if I set `useHash = true`, then I would be able to parse it, provided that I rewrite the URL above as follows:
```
http://myhost/myproject/#!?check=1
```

**Comment by krasimir** [comment]
Hey @lbssousa,

I added a unit test but no luck reproducing your issue. I created a simple page with the following code:
```js
var router = new Navigo('http://home.dev/Krasimir/_tests/navigo/96/', false, '#!');

  router.on(function (query) {
    console.log('Query string: ' + query);
  }).resolve();
```
And when I opened http://home.dev/Krasimir/_tests/navigo/96/?test=1 I got:
![image](https://cloud.githubusercontent.com/assets/528677/24524570/be462574-159f-11e7-9741-a2e02168f087.png)

So I'm not able to replicate the issue locally. Maybe if you provide an example. Also let me know OS and browser version. Do you use the latest Navigo release?

**Comment by lbssousa** [comment]
Hmmm... Sorry, but I wasn't too accurate on my issue description.

In fact, if I initialize the router with an explicit root URL (as in your test case), I'm able to catch the query string. The real issue occurs when I initialize the router with a default root URL (passing `null` as the first argument). Taking your example code, it should be
```js
var router = new Navigo(null, false, '#!');

  router.on(function (query) {
    console.log('Query string: ' + query);
  }).resolve();
```

In this case, I'm getting only an empty string for `query`.

**Comment by krasimir** [comment]
I finally reproduced the bug. Fixed in `4.5.1`. Can you please try and let me know if it works on your machine.

**Comment by lbssousa** [comment]
Yes! I've just upgraded my navigo version and now it parses the query string correctly!

Thank you very much!
